gtk/src/OutputEditor.hh

```diff
@@ -74,4 +74,4 @@ public:
 	virtual void setLanguage(const Config::Lang& /*lang*/) {}
 };
 
-#endif // OUTPUTEDITOR_HH
+#endif // OUTPUTEDITOR_HH
```

Comment:
Can you configure your editor to not strip the trailing line breaks?

Reply:
Hmm... I indeed used Gnome Builder for the first time to fix this issue. I must admit I'm not familiar with its UI. I could not find the setting that controls "stripping the trailing line breaks"... I tried to google it and here is what I've found:

https://gitlab.gnome.org/GNOME/meld/-/issues/134

"However, neither gedit nor Builder expose it as a visible preference..."

Reply:
In case just selectively pick the hunks you want to commit with `git add -p` and omit the whitespace changes from the commit.

Reply:
I tried to use `git add -p`... I'm not sure whether it helped...

Reply:
It looks OK now...